Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -781,10 +813,19 @@ - (void)loadView
         [[_banner centerXAnchor] constraintEqualToAnchor:self.view.centerXAnchor],
 #endif
         [[_stackView leadingAnchor] constraintEqualToAnchor:margins.leadingAnchor],
+#if PLATFORM(VISION)
```

Comment:
Should this be `ENABLE(LINEAR_MEDIA_PLAYER)` to match the other changes in this patch?